Base/usr/share/man/man1/cp.md

```diff
@@ -0,0 +1,36 @@
+## Name
+
+cp - copy files
+
+## Synopsis
+
+```**sh
+$ cp [options...] <source...> <destination>
+```
+
+## Description
+
+`cp` copies files specified in `source` to `destination`.
+
+If there are several `sources`, the directory `destination` is created an all files specified in `sources` are copied into that directory. Otherwise, the `source` file is copied as `destination` file. If the file exists, it is overridden. If `destination` directory exists and there is only one `source`, the `source` file is copied into `destination` directory.
```

Comment:
```suggestion
If there are several `sources`, the directory `destination` is created and all files specified in `sources` are copied into that directory. Otherwise, the `source` file is copied as `destination` file. If the file exists, it is overridden. If `destination` directory exists and there is only one `source`, the `source` file is copied into the `destination` directory.
```